File: Mobile/media_tracker_test/lib/models/tmdb/tmdb_account.dart
class TMDBAccount {
  final String username;
  final int id;

  TMDBAccount({required this.username, required this.id});

  factory TMDBAccount.fromJson(Map<String, dynamic> json) {
    return TMDBAccount(
      username: json['username'],
      id: json['id'],
    );
  }
}

File: Desktop/media-tracker-desktop/media-tracker-desktop/Models/TMDB/TMDB_Account.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace media_tracker_desktop.Models.TMDB
{
    public class TMDB_Account
    {
        [JsonProperty("username")]
        public string Username { get; } = string.Empty;

        [JsonProperty("id")]
        public int ID { get; }

        [JsonConstructor]
        public TMDB_Account(string username, int id)
        {
            this.Username = username;
            this.ID = id;
        }
    }
}
